ironfish/src/utils/currency.ts

```diff
@@ -69,6 +69,20 @@ export class CurrencyUtils {
 
     return ore
   }
+
+  static renderBalance(amount: bigint | string, assetIdentifier?: string): string {
+    if (typeof amount === 'string') {
+      amount = this.decode(amount)
+    }
+
+    const balance = FixedNumberUtils.render(amount, 8)
+
+    if (assetIdentifier) {
```

Comment:
What do you think about displaying $IRON if the native asset identifier? Might be able to fallback on renderIron() in this file or something